### ISSUE TITLE
Support .net core 2.0

### ISSUE THREAD

**Issue by Domitnator** [Kentico/delivery-sdk-net#95]
Hi Guys,

It would be great if the SDK could be upgraded to .net core 2.0!

Kind Regards,

Alfred

**Comment by petrsvihlik** [comment]
Hi @Domitnator, 
the SDK targets `netstandard1.3` making it compatible with [.NET Core 1.0 and above, and .NET 4.6 and above](https://docs.microsoft.com/en-us/dotnet/standard/net-standard). 
Do you know of any particular functionality in .NET Core 2 (or rather .NET Standard 2.0) that this SDK could benefit from?
We'd like to keep the target versions as low as possible as it makes it compatible with wider specter of platforms and devices.

**Comment by Domitnator** [comment]
Hi Petr,

I just realized that i posted this question in the wrong repository. I should have been on the https://github.com/Kentico/cloud-boilerplate-net

I had to many tabs open :)

In the meanwhile I upgraded the project ("ASP.NET Core MVC with Kentico Cloud") myself to .net core 2.0. Still struggling with one thing (iisrewrite). I will try to commit my changes next week so other people can benefit from it.
